test/ShapeCrawler.Tests.Unit/PictureTests.cs

```diff
@@ -219,6 +219,94 @@ public void Crop_setter_sets_expected_values(string expectedFrameStr)
         actual.Should().Be(expected);
     }
 
+    [Test]
+    [SlideShape("059_crop-images.pptx", 1, "None", "Rectangle")]
+    [SlideShape("059_crop-images.pptx", 1, "RoundedRectangle", "RoundedRectangle")]
+    [SlideShape("059_crop-images.pptx", 1, "TopCornersRoundedRectangle", "TopCornersRoundedRectangle")]
+    [SlideShape("059_crop-images.pptx", 1, "Star5", "Star5")]
+    public void Picture_geometry_getter_gets_expected_values(IShape shape, string expectedStr)
+    {
+        // Arrange
+        var expected = (Geometry)Enum.Parse(typeof(Geometry),expectedStr);
```

Comment:
Can be simplified with `Enum.Parse<Geometry>(expectedStr)`?

Reply:
That fails when compiling under net standard 2.0, so wouldn't pass release mode build, sadly.